source/_integrations/senz.markdown

```diff
@@ -0,0 +1,18 @@
+---
+title: nVent RAYCHEM SENZ
```

Comment:
Why doesn't the title match the domain?

Reply:
I use the full name (Company + Brand + Series) as the manufacturer does on its site, packaging, marketing, manual, etc. Because the company changes its name about every 5 years, I chose to only use the model as the domain.

Reply:
We generally don't name integrations after a series, just brands. Does this API only serve this series? or the whole brand.

Reply:
* The API is specific to this series, and [the API documentation](https://api.senzthermostat.nvent.com) refers to it as "SENZ RestAPI".
* There are other series under the nVent RAYCHEM name that use bluetooth and modbus APIs.
* There are other brands under the nVent name, such as Nuheat with their own API and integration: https://www.home-assistant.io/integrations/nuheat/